mobile-browser-based-version/server/src/logic/federated/handlers.js

```diff
@@ -269,7 +269,6 @@ export function selectionStatus(request, response) {
   if (!clients.has(id)) {
     return _failRequest(response, type, 401);
   }
-
   if (!tasksStatus.has(task)) {
     return _failRequest(response, type, 404);
```

Comment:
For the errors, should we maybe add an error.js file, and create variables there:
- Make errors easier to understand in code + also in log output